column/api/backend/cache/local_mem.py

```diff
@@ -0,0 +1,51 @@
+from collections import deque  # noqa
+import threading
+
+import store
+
+
+# Shared dictionaries for LocalMemoryCache
+_stores = {}
+_key_queues = {}
+_locks = {}
+
+MAX_STORE_SIZE = 50
+
+
+class LocalMemoryStore(store.Store):
+
+    def __init__(self, name):
+        self._store = _stores.setdefault(name, {})
+        self._key_queue = _key_queues.setdefault(name, deque([]))
+        lock = _locks.setdefault(name, threading.Lock())
+        super(LocalMemoryStore, self).__init__(name, lock)
+
+    def _has_key(self, key):
+        if key in self._store:
+            return True
+        return False
+
+    def _add(self, key, value):
+        self._save(key, value)
+        self._key_queue.append(key)
+
+    def _del(self, del_key):
```

Comment:
"del_key" -> "key" for consistency with its parent class (or the other way round, whichever easier)